### ISSUE TITLE
Add int as identifier to currency and language table

### ISSUE THREAD

**Issue by Bjoern3003** [PHPfox-Official/phpfox-v4-issues#2758]
Please add integer to currency table as unique identifier. varchar type is no good id to work with third party applications.

**Comment by harrison05** [comment]
Our team will check your request. 

P/s: 3rd app can modify Currency table to add Unique ID, just drop current primary key "currency_id" and set a new "id" column with auto_increament as a primary key.

Regards.

**Comment by Bjoern3003** [comment]
Yes, I know, that we can modify all the databases with their install routine. But I think, this has to set by default.

**Comment by Bjoern3003** [comment]
AlLso add the same to the language table.

**Comment by harrison05** [comment]
> AlLso add the same to the language table.

We will check it too.
Regards.
